packages/compass-components/src/constants/mongodb-ace-theme-query.ts

```diff
@@ -111,6 +111,7 @@ background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZg
 const mongodbAceThemeQueryCssText = `
 .ace-mongodb-query .ace_scroller {
 margin: 0px 6px;
+line-height: 14px;
```

Comment:
Are all these hardcoded `px` all right here? No need to worry in this PR but feels a bit weird that we are not using variables, don't we have other components/texts that are meant to have the same fonts, spacings, colors and border-radius?

Reply:
This file has a lot of them, I've added updating the ace themes to the new brand in the tech design. I'm thinking we can update this theming to use variables like palette and spacing then.